.github/workflows/release.yml

```diff
@@ -0,0 +1,55 @@
+name: WDL Release Tests
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+    types:
+      - opened
+      - synchronize
+      - ready_for_review
+      - labeled
+
+jobs:
+  test:
+    name: dxCompiler Integration Tests (Staging)
+    if: contains(github.event.pull_request.labels.*.name, 'integration')
```

Comment:
We don't want to run the release and large tests with every commit to a PR labelled 'integration' . So how about for the release a label 'release' needs to be used?

Perhaps there is even a better trigger available - one where we don't need to create a PR to run a release.